src/Angor/Shared/Services/RelayService.cs

```diff
@@ -271,11 +271,14 @@ public Task<string> DeleteProjectAsync(string eventId, string hexPrivateKey)
             return Task.FromResult(deleteEvent.Id);
         }
         
-        private static NostrEvent GetNip78NostrEvent( string content)
+        private static NostrEvent GetNip3030NostrEvent( string content)
         {
+            // https://github.com/block-core/nips/blob/peer-to-peer-decentralized-funding/3030.md
+
             var ev = new NostrEvent
             {
-                Kind = NostrKind.ApplicationSpecificData,
+                //Kind = NostrKind.ApplicationSpecificData,
+                Kind = (NostrKind)3030,
```

Comment:
Did you change all the filteres on the requests as well?

Reply:
I dont think we use that as filter but I will check